packages/ember-auto-import/ts/analyzer.ts

```diff
@@ -118,36 +73,23 @@ export default class Analyzer extends Plugin {
     return this.modules;
   }
 
-  async build() {
-    await this.setupParser();
-    this.getPatchset().forEach(([operation, relativePath]) => {
-      let outputPath = join(this.outputPath, relativePath);
-
+  async build(...args: unknown[]) {
+    await super.build(...args);
```

Comment:
Whats the TL;DR of using `Funnel` as the base class?  It sorta hurts my brain to think about how that works. It seems like we would end up potentially end up double walking the input right?

Reply:
Funnel with no options is optimized to be the fastest pass through it can, including symlinking the whole tree when it can.

Reply:
It would be nice if broccoli had a first-class read-only primitive but I don't think it does. So a read-only transform like ours is obligated to propagate changes forward and needs to do so efficiently.

Reply:
@ef4 oh I get it thanks. That is a good "hack",  and makes sense in this context.